types/overlayscrollbars/overlayscrollbars-tests.ts

```diff
@@ -0,0 +1,2 @@
+function test_init() {
```

Comment:
Something tells me that this isn't how `OverlayScrollbars` is used 😄 

Could you write a meaningful example of how to use the library of the test?

Reply:
I've added the missing tests and improved the definitions a bit